### ISSUE TITLE
Redirect request: "Azure AD Connect sync: Prevent accidental deletes"

### ISSUE THREAD

**Issue by M365SupportJapan** [MicrosoftDocs/feedback#704]
**🛑 IMPORTANT**: You can get your feedback addressed faster if you **use the comment section for the article in which you encountered a problem**.

**Link to article:**
[EN-US]
https://docs.microsoft.com/en-us/azure/active-directory/active-directory-aadconnectsync-feature-prevent-accidental-deletes

[JA-JP]
https://docs.microsoft.com/ja-jp/azure/active-directory/active-directory-aadconnectsync-feature-prevent-accidental-deletes

**Problem:**
Please consider to redirect the URL to the new one of the article titled "Azure AD Connect sync: Prevent accidental deletes"

Old URL - Returns 404 error.
[EN-US]
https://docs.microsoft.com/en-us/azure/active-directory/active-directory-aadconnectsync-feature-prevent-accidental-deletes

[JA-JP]
https://docs.microsoft.com/ja-jp/azure/active-directory/active-directory-aadconnectsync-feature-prevent-accidental-deletes

URL at present: 
[EN-US]
https://docs.microsoft.com/en-us/azure/active-directory/hybrid/how-to-connect-sync-feature-prevent-accidental-deletes

[JA-JP]
https://docs.microsoft.com/ja-jp/azure/active-directory/hybrid/how-to-connect-sync-feature-prevent-accidental-deletes

**Comment by M365SupportJapan** [comment]
Hi Jian-san,

Thank you for reaching me out.

Checked says “Identity synchronization Error Report” email provided the old URL.
We also escalated it to the engineering team via IcM though, it would be nice if the old URL redirect to the new URL.

Online: Showing incorrect URL link in "Identity synchronization Error Report" email
https://icm.ad.msft.net/imp/v3/incidents/details/85726584/home

Thank you all for your consideration in advance,
Mizuho

Mizuho Takahashi | Technical Advisor | Microsoft 365 Support Japan
